src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,13 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService userService = new UserService();
+        User user = userService.findByEmail(email);
+
+        if (userService.findByEmail(email) != null) {
+            return user.getPassword().equals(password);
+        }
```

Comment:
You already have user field, why do you duplicate code of user inside if(......)?
You can simplify it to one line in return